top-level-functions.typ

```diff
@@ -0,0 +1,8 @@
+= Top Level Functions
+
+Our system must be able to perform these functions in order to accomplish our mission
++ Deliver messages between Earth and Mars: The primary function of the C3 system is to facilitate the exchange of messages and data across vast interplanetary distances. This system ensures that crucical information can be transmitted reliably to and form Earth.
++ Command manned and unmanned missions from Earth: The C3 system must be able to send commands to and receive data from manned and unmanned missions to Mars. This includes the ability to send commands to rovers, satellites, and other equipment on the Martian surface.
++ Allow for interpersonal communication on the colony: The C3 system must provide a means for colonists to communicate with each other and with Earth. This includes voice, video, and text communication.
++ Protocol Management: The C3 system should adhere to established communication protocols for interoperability with existing infrastructure on Earth.
++ Security and Encryption: The C3 system must implement security measures to protect against unauthorized access or tampering of transmitted missions. This includes encryption of data.
```

Comment:
These are requirements, not top level functions

Reply:
I think you can argue for both. I definitely agree that they are leaning more towards requirements now that I look again though. 

Do you think our top level functions are a little bare bone or are they sufficient?